c/chosen/chosen_rhel_8_4.sh

```diff
@@ -0,0 +1,31 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : chosen
+# Version       : 1.0.0
+# Source repo   : https://github.com/harvesthq/chosen/archive/1.0.0.zip
+# Tested on     : UBI 8
+# Script License: Apache License, Version 2 or later
+# Maintainer    : sachin.kakatkar@ibm.com
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#To Run: ./chosen_rhel_8_4.sh
+dnf install git wget npm make gcc-c++ bzip2 unzip ruby -y
+wget https://github.com/harvesthq/chosen/archive/1.0.0.zip
```

Comment:
Never download the ZIP/tar files for build and validation.
Please use the `git clone` for the repo and `git checkout` for the Version tag/branch.

@seth-priya : Correct me if I am wrong on above comment.

Use package version into a variable and take it from command line parameter. If it is not provided, you can take one version (currently on which version we are working) as default version for build and validation.

Code should be as below :
```
PACKAGE_VERSION=${1:-1.0.0}
git clone https://github.com/harvesthq/chosen
cd chosen
git checkout $PACKAGE_VERSION
```

This is a generic comment and applicable for all the build-scripts.

Reply:
1.0.0 is not available in git hub repo.

Reply:
I can see version is available at : https://github.com/harvesthq/chosen/tree/1.0.0

I tried checkout myself and it is working fine.
```
~]# git clone https://github.com/harvesthq/chosen; cd chosen
Cloning into 'chosen'...
remote: Enumerating objects: 7464, done.
remote: Total 7464 (delta 0), reused 0 (delta 0), pack-reused 7464
Receiving objects: 100% (7464/7464), 2.98 MiB | 23.32 MiB/s, done.
Resolving deltas: 100% (4595/4595), done.

 chosen]# git checkout 1.0.0
Note: switching to '1.0.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b34a5d8 Updating to 1.0
```

![image](https://user-images.githubusercontent.com/69784217/147684114-06a99baf-eac5-4c82-81d1-777a2fbcd600.png)